string DroneID
string hostel_to
---
bool success
